hack/components/bump-linux-bridge.sh

```diff
@@ -39,7 +46,7 @@ RUN \
 WORKDIR ${LINUX_BRIDGE_PATH}
 RUN GOFLAGS=-mod=vendor ./build_linux.sh
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal
+FROM registry.access.redhat.com/ubi8/ubi-minimal AS final
```

Comment:
please drop AS final, it is not used
prone theoretically for mistakes if someone adds layers below that

Reply:
Done.